pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -2662,7 +2662,7 @@ public void getDispatchRate(@Suspended final AsyncResponse asyncResponse,
     }
 
     @POST
-    @Path("/{tenant}/{namespace}/{topic}/dispatchRate")
+    @Path("/{tenant}/{namespace}/{topic}/{resetMode}/dispatchRate")
```

Comment:
This is a breaking API change.
Can we use a query string parameter?

Reply:
Good suggestion :)
Done